backend/models/Notice.model.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const noticeSchema = new mongoose.Schema({
+    title: {
+        required: true,
+        type: String,
+        trim: true
+    },
+    createdAt: {
+        required: true,
+        type: Date,
+        default: Date.now()
+    },
+    description: {
+        required: true,
+        type: String,
+        trim: true
+    },
+    pdfPath: {
```

Comment:
Fingers crossed for cloud services? or we'll have it in the deployment itself?

Reply:
cloud services will be great . . . can't rely on deployment!

Reply:
Maybe we should talk with the authorities about this before planning it out.

Reply:
Point, will figure it out in meet